RxKeyboard.podspec

```diff
@@ -11,8 +11,8 @@ Pod::Spec.new do |s|
   s.frameworks       = 'UIKit', 'Foundation'
   s.requires_arc     = true
 
-  s.dependency 'RxSwift', '>= 4.0.0'
-  s.dependency 'RxCocoa', '>= 4.0.0'
+  s.dependency 'RxSwift', '>= 4.1.2'
+  s.dependency 'RxCocoa', '>= 4.1.2'
```

Comment:
Do we need RxSwift 4.1.2 for Swift 4.1? RxSwift 4.0.0 seems to be still work with Swift 4.1. Not all of RxKeyboard users might have upgraded RxSwift dependency to the latest, so I think we would better keep this dependency if not necessary.

Reply:
I think not ,

No problem man, I will change it! 

Thank you!